File: WebContent/js/MemberFindIdAction.js
/**
 * 
 */

		var findIdCheck=false;

function FindId() {
	var forward = new ActionForward();
	
	var mDao = new MemberDAO();
	var email = request.getParameter("member_email");
	var name = request.getParameter("member_nickname");
	var  out = response.getWriter();
	
	var id = mDao.findId(email, name);
		
		
		if(id!=null) {
			$.ajax({
				url : contextPath + "/member/MemberCheckIdOk.me?id=" + id,

				type : 'get',
				dataType : 'text',
				success : function(data) {

					if (data.trim() == 'ok') {
						$("#findIdCheckText").text("당신의 아이디는"+id+"입니다.");
						findIdCheck = true;
					} else {

						$("#findIdCheckText").text("없는 회원입니다.");
						findIdCheck= false;
					}
				},
				error : function() {
					console.log("오류");
				}
			})
		}
				
			
			

	}

File: src/com/aboplate/app/member/MemberFindIdAction.java
package com.aboplate.app.member;

import java.io.PrintWriter;

import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;

import com.aboplate.action.Action;
import com.aboplate.action.ActionForward;
import com.aboplate.app.member.dao.MemberDAO;

public class MemberFindIdAction implements Action{
	
	@Override
	public ActionForward execute(HttpServletRequest request, HttpServletResponse response) throws Exception {
		request.setCharacterEncoding("UTF-8");
		response.setCharacterEncoding("UTF-8");
		
		ActionForward forward = new ActionForward();
		MemberDAO mDao = new MemberDAO();
		String email = request.getParameter("memberEmail");
		String name = request.getParameter("memberName");
		
		String id = mDao.findId(email, name);
		
		PrintWriter out = response.getWriter();
		if(id != null) {
			System.out.println(id);
			response.setContentType("text/html; charset=UTF-8");
			out.println("<script>");
			out.println("alert('당신의 아이디는"+id+"입니다.'); location.href='login.jsp'; ");
			out.println("</script>");
			out.close();
			}else {
			response.setContentType("text/html; charset=UTF-8");
			out.println("<script>");
			out.println("alert('등록된 아이디가 없습니다. 다시 시도해주세요.'); history.back(); ");
			out.println("</script>");
			out.close();
		}
		forward.setRedirect(true);
		return forward;
	}
}
